### ISSUE TITLE
Two question about ScaleDepth

### ISSUE THREAD

**Issue by huangliqwe2020** [RuijieZhu94/mmdepth#6]
Dear author, your work on estimating metric depth by combining semantics from CLIP is truly impressive and admirable! But I still have doubts about how to implement metric depth in the paper: (1) Does CLIP have the ability to predict scene geometry information through text? (2) Can we using the semantic features provided by CLIP alone to supervise scale queries to achieve estimation of metric scale. I would greatly appreciate your answer!

**Comment by RuijieZhu94** [comment]
Thank you for your interest in our work!

(1) The capabilities of CLIP can be considered from two aspects. On one hand, there's the feature extraction capability of its pre-trained visual layers. Although CLIP’s pre-training focuses on semantic information, we believe that with appropriate fine-tuning, it can robustly extract depth features, as demonstrated by the ARDE module in our paper. On the other hand, CLIP's scene understanding ability includes alignment with text. Through suitable training, we can associate text with scale, enabling it to generalize scale information across pre-trained scene categories, as shown by the SASP module in our paper.

(2) Sure, it is possible and might even yield better results. However, this was not our primary goal. We aimed to associate it with text to leverage textual supervision and enhance its scene category recognition. As a result, in unseen scenes with similar scene categories, our method can also predict accurate scene scales.

**Comment by huangliqwe2020** [comment]
Thanks for your answer!
